projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/props.tsx

```diff
@@ -0,0 +1,19 @@
+
+
+export interface IJobPathProp {
+    /**
+     * Current Job path or the path to job's parent directory
+     */
+    jobPath: string;
```

Comment:
it just seems strange from OOP perspective, having an interface  field. 
Interface is a stateless abstraction
https://peps.python.org/pep-0245/

Reply:
Fields in interfaces are allowed in typescript. If it looks two weird, maybe @duyguHsnHsn can replace it with a type alias instead. They're effectively the same thing, minus extensibility.

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

Reply:
changed with type.